.github/workflows/build.yaml

```diff
@@ -35,15 +35,32 @@ env:
   THELMA_RELEASE_BUCKET: thelma-releases
   # Version of gcloud to use for workflow steps that require it
   SETUP_GCLOUD_VERSION: '0.3.0'
+  # macOS release signing cert
+  THELMA_MACOS_CERT: ${{ secrets.THELMA_MACOS_CERT}}
+  # macOS release signing cert password
+  THELMA_MACOS_CERT_PWD: ${{ secrets.THELMA_MACOS_CERT_PWD }}
+  # Apple Developer application password
+  THELMA_MACOS_APP_PWD: ${{ secrets.THELMA_MACOS_APP_PWD }}
 jobs:
   build:
     runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.tag.outputs.tag }}
     steps:
       - name: Checkout current code
         uses: actions/checkout@v2
         with:
           token: ${{ secrets.BROADBOT_TOKEN }}
 
+      - name: Cache build directory
```

Comment:
so just checking my understanding, this isn't actually caching as much as "save contents of build output directory and make available for the notarization job"?